packages/lwc-language-server/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
     "compilerOptions": {
-        "target": "es2017",
+        "target": "es2018",
```

Comment:
why did you go with 2018?

https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-20